ignition.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Since `ignition.json` is automatically generated, it should be `.gitignore`ed, I think. Otherwise user will find a modified file in the repo after running make when they updated `flatcar-linux-config.yml` with their own ssh key for example.

Reply:
done